mlrun/runtimes/databricks_job/databricks_wrapper.py

```diff
@@ -209,4 +209,8 @@ def print_status(run: Run):
     logger.info(f"logs:\n{run_output.logs}")
     run_output_dict = run_output.as_dict()
     run_output_dict.pop("logs", None)
-    logger.info(f"Run output and metadata:\n{run_output_dict}\n")
+    context.log_artifact(
+        f"databricks_run_metadata_{context.uid}",
```

Comment:
```suggestion
        f"databricks_run_metadata",
```
You should not add the `uid` to the artifact key. It will be added as the `tree`, or `producer_uid` anyway since this is logged from an execution context. You don't have to worry about the artifact being overridden.